pkg/system/phase2_creating.go

```diff
@@ -889,6 +890,20 @@ func (r *Reconciler) UpgradeMigrateDB() error {
 			return err
 		}
 		if phase == "" {
+			// when starting - restart the db pod. This is a fix for https://bugzilla.redhat.com/show_bug.cgi?id=1922113
+			r.Logger.Info("attempt to delete noobaa-db-0")
```

Comment:
Isn't it getting the noobaa-db-0 and not deleting?

Reply:
I get the object first before deleting it

Reply:
Yes, it is getting the object but the printing can mislead.
```suggestion
			r.Logger.Info("attempt to delete noobaa-db-0, getting the pod")
```